vud_widget/src/Annotation/VoteUpDownWidget.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\vud_widget\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Vote Up/Down Field item annotation object.
+ *
+ * @see \Drupal\vud_widget\Plugin\VoteUpDownFieldManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class VoteUpDownWidget extends Plugin {
```

Comment:
This looks fine, but:
* again let's move it to vud module,
* please also include a couple more data members: widgetTemplate, and votesTemplate, see e.g. `widgets/plain/plain.inc`. This may change if we can replace `vud_pseudo_theming()` with real theme templating, but let's start with the same D7 approach.